spec/Section 5 -- Validation.md

```diff
@@ -556,7 +556,7 @@ fragment safeDifferingArgs on Pet {
 ```
 
 However, the field responses must be shapes which can be merged. For example,
-scalar values must not differ. In this example, `someValue` might be a `String`
+scalar types must not differ. In this example, `someValue` might be a `String`
```

Comment:
You did not apply the suggested change. Read still scalar but meant is indeed a leaf type.

```suggestion
leaf types must not differ. In this example, `someValue` might be a `String`
```